WBooks/Screens/MainViewController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  MainViewController.swift
+//  WBooks
+//
+//  Created by Nacho 2 on 20/01/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class MainViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    private let _view: MainView = MainView.loadFromNib()!
+    
+    var booksArray : Array = [["image": "img_book2", "title": "The best book in the world", "subtitle": "Peter Sjermstrom"], ["image": "img_book3", "title": "A little bird told me", "subtitle": "Timpthy Cross"], ["image": "img_book4", "title": "When the doves desappeared", "subtitle": "Sofi Oksanen"]]
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        booksArray.count
+    }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        100
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "CustomBookCell", for: indexPath) as! CustomBookCell
```

Comment:
We have some helpful functions in wolmo core for this. 
Check register and dequeue funcs [here](https://github.com/Wolox/wolmo-core-ios/blob/master/WolmoCore/Extensions/UIKit/UITableView.swift)

Reply:
nice!